scripts/create_newcase

```diff
@@ -171,7 +171,7 @@ def _main_func(description):
         user_mods_dir, user_compset, pesfile, \
         user_grid, gridfile, srcroot, test, ninst, walltime, queue, \
         output_root, script_root, run_unsupported, \
-        answer, input_dir = parse_command_line(sys.argv, cimeroot, description)
+        answer, input_dir, = parse_command_line(sys.argv, cimeroot, description)
```

Comment:
Not even sure how that got there. I don't remember doing that, but clearly I must have.